### ISSUE TITLE
DESIGN:  Username vs. E-mail vs. Login

### ISSUE THREAD

**Issue by catrobson** [fabric8-ui/fabric8-ux#650]
Confusion about email vs. user name vs. login -- you have to enter this correctly to reset tenet.  We're using different things everywhere.

Come up with standard terminology and usage that we should have in our system.  Document where these should be updated for a dev story.

**Comment by mceledonia** [comment]
### Current Usage:

- **Account Login:** Email address or other Red Hat Login ID
- **Reset Tenant Modal:** Openshift.io Username
- **Edit Profile:** Email

### Proposed Usage:

- **Account Login:** Email
- **Reset Tenant Modal:** Email
- **Edit Profile:** Email

_This proposal makes the assumption that the vast majority of users, if not all, will have an email associated with their account that they can log in to. In the event we must offer both a username and email login, see below..._

### Backup Usage

- **Account Login:** Username or Email
- **Reset Tenant Modal:** Username
- **Edit Profile:** Email

_If logging in with username is a must-have, we can consolidate the text in both the login page and the resent tenant modal to the above in order to make things feel less convoluted. The problem with the current design may just be the extraneous text, i.e. "Red Hat Login ID" and "Openshift.io Username."_

@catrobson

**Comment by Essjaysee** [comment]
In the case of my own account, which I registered for from scratch, my username is actually my email address.  From my rough understanding, that may not be the case for all users (and I'm not super clear on the circumstances that would lead that to be the case).  

One possibility could be to present the text of what we consider to be their username and then they type it -- it's the typing that we're interested in.

**Comment by catrobson** [comment]
We also need to figure out what to show in the top right of the UI (Right now it is full name), and the top left of the UI when the user is shown (right now it is username, which is also used in the URL).  I don't think changing the top left to e-mail makes sense, nor the URL.  Showing the full user's name in top right, what do you think about that?

**Comment by mceledonia** [comment]
@catrobson I agree, having full name in the top right and the username in top left (when not in a space) makes sense to me. I can update these screens to show that in action.
